accessibility-checker-engine/help-v4/en-US/aria_activedescendant_valid.html

```diff
@@ -67,7 +68,7 @@ <h3 id="group_label_1">Pizza Crust</h3>
    </ul> 
 ``` 
    
-* Example code includes material copied from or derived from ARIA practices - Radio Group Example Using aria-activedescendant. Copyright © 2018-2019 W3C® (MIT, ERCIM, Keio, Beihang).
+* Example code includes material copied from or derived from ARIA practices - Radio Group Example Using aria-activedescendant. Copyright © 2018-2019 W3C® (MIT, ERCIM, Keio, Beijing ).
```

Comment:
Beihang was correct.
```suggestion
* Example code includes material copied from or derived from ARIA practices - Radio Group Example Using aria-activedescendant. Copyright © 2018-2019 W3C® (MIT, ERCIM, Keio, Beihang).
```